lib/screens/add_plate_weights.dart

```diff
@@ -0,0 +1,118 @@
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:wger/providers/plate_weights.dart';
+
+class AddPlateWeights extends StatefulWidget {
+  const AddPlateWeights({super.key});
+
+  @override
+  State<AddPlateWeights> createState() => _AddPlateWeightsState();
+}
+
+class _AddPlateWeightsState extends State<AddPlateWeights> {
+
+  @override
+  Widget build(BuildContext context) {
+    return Consumer<PlateWeights> (
+      builder:(context,plateProvider,child)=> Scaffold (
+        appBar: AppBar (
+          title: const Text('Enter Custom Plate Weights'),
+        ),
+        body: Column (
```

Comment:
make sure there is enough margin on the right side of the screen as well (or wrap around if there's not enough space). Also I would name the dropdown something like "preferred unit". If you want, you can extract this info from the profile from the UserProvider (the isMetric).

![Bildschirmfoto 2024-12-12 um 13 02 35](https://github.com/user-attachments/assets/e1e22890-3b30-4c35-8747-b0f0ccf81fae)